docs/_src/language/basic.md

```diff
@@ -184,13 +264,15 @@ explore airports
 | reduce
   state
   airport_count
-  by_facility is (reduce
+  by_facility is (reduce top 2
     fac_type
     airport_count
   )
 ```
 
-## Nesting within Nested Queries
+Here we can see that the `by_facility` column of the output table contains nested subtables on each row. When interpreting these inner tables, all of the dimensional values from outer rows still apply to each of the inner rows.
+
+Queries can also be nested to any depth, allowing for rich, complex output structures.
```

Comment:
"Infinitely" is actually confusing to me, because it sounds like you're talking about recursion, which we don't allow today, and is also a difficulty concept. 

Maybe something like "Queries can also be deeply nested, allowing for rich, complex output structures. A query may always include another nested query, regardless of depth."

Reply:
I don't totally get the connection to recursion here--I think of recursion in SQL referring to a CTE (which we do support!). One can recurse infinitely just as one can nest infinitely; I like "infinitely" because it's concise and true.  But if you feel really strongly about this I'm not dead set on it.

Reply:
When I say "recurse" here, I mean defining a query that nests itself, and therefore is infinite.... maybe I'm the only one to think that...

Compromise: "Queries can be nested infinitely, allowing for rich, complex output structures. A query may always include another nested query, regardless of depth."